shopelectro/admin.py

```diff
@@ -238,10 +238,11 @@ class OrderAdmin(mixins.PermissionsControl):
 
     add = False
     inlines = [PositionInline]
-    list_display = ['name', 'email', 'phone', 'address', 'city', 'payment_type', 'paid']
+    list_display = ['id', 'name', 'email', 'phone', 'total_price', 'payment_type', 'paid']
     search_fields = ['name', 'email', 'phone']
     list_display_links = ['name']
 
+
```

Comment:
this blank line is not required

Reply:
@artemiy312 , that's what pycharm told me about it:
![image](https://user-images.githubusercontent.com/3526796/44616362-1273cc00-a857-11e8-83c3-7db859e0b937.png)

It's strange, coala should check it. This issue was fixed at version 2.2.0 of pycodestyle. But our coala uses version pycodestyle==2.3.1
https://github.com/PyCQA/pycodestyle/issues/400